src/Blocks/components/conditional-tags/components/conditional-tags-options.js

```diff
@@ -89,217 +64,241 @@ export const ConditionalTagsOptions = (attributes) => {
+							onClick={() => {
+								conditionalTagsRules[1][parent][index + 1] = [formFields?.[0]?.value ?? '', CONDITIONAL_TAGS_OPERATORS.IS, ''];
+								setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: [...conditionalTagsRules] });
+							}}
+							className='es-rounded-1'
+						>
+							{__('AND', 'eightshift-forms')}
+						</Button>
+					}
+
+					<Button
+						icon={icons.trash}
+						onClick={() => {
+							conditionalTagsRules[1][parent].splice(index, 1);
+
+							if (conditionalTagsRules[1][parent].length === 0) {
+								conditionalTagsRules[1].splice(parent, 1);
+							}
+							setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: [...conditionalTagsRules] });
+						}}
+						label={__('Remove', 'eightshift-forms')}
+						className='es-ml-auto es-rounded-1!'
+					/>
+				</div>
 			</>
 		);
 	};
 
-	const optionsItem = formFields?.find((item) => item.value === conditionalTagsParentName)?.subItems ?? [];
+	const rulesCount = conditionalTagsRules?.[1]?.length && conditionalTagsRules?.[1]?.flat()?.length;
 
 	return (
-		<PanelBody>
-			<IconToggle
-				icon={icons.conditionalVisibility}
-				label={__('Conditional visibility', 'eightshift-frontend-libs')}
-				checked={conditionalTagsUse}
-				onChange={(value) => {
-					setAttributes({ [getAttrKey('conditionalTagsUse', attributes, manifest)]: value });
-
-					if (!value) {
-						setAttributes({ [getAttrKey('conditionalTagsAction', attributes, manifest)]: undefined });
-						setAttributes({ [getAttrKey('conditionalTagsLogic', attributes, manifest)]: undefined });
-						setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: undefined });
-					} else {
-						setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: [] });
-					}
-				}}
-				noBottomSpacing={!conditionalTagsUse}
-				additionalClasses='es-font-weight-500'
-			/>
-
-			<Section showIf={conditionalTagsUse} noBottomSpacing>
+		<>
+		{formFields?.length < 1 ?
+			<PanelBody>
 				<Control
-					icon={icons.conditionH}
-					label={__('Rules', 'eightshift-forms')}
-					// Translators: %d refers to the number of active rules
-					subtitle={conditionalTagsRules?.length > 0 && sprintf(__('%d added', 'eightshift-forms'), conditionalTagsRules.length)}
+					icon={icons.conditionalVisibility}
+					label={__('Conditional visibility', 'eightshift-forms')}
+					additionalLabelClasses='es-font-weight-500'
 					noBottomSpacing
-					inlineLabel
 				>
-					<Button
-						variant='tertiary'
-						onClick={() => setIsModalOpen(true)}
-						className='es-rounded-1.5 es-w-9 es-h-center es-font-weight-500'
-					>
-						{conditionalTagsRules?.length > 0 ? __('Edit', 'eightshift-forms') : __('Add', 'eightshift-forms')}
-					</Button>
+					<IconLabel
+						icon={icons.warningFillTransparent}
+						label={__('Feature unavailable', 'eightshift-forms')}
+						subtitle={__('No fields have a name set', 'eightshift-forms')}
```

Comment:
Wording feels kind of strange here, I'd suggest something but I'm lacking context